traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 8.9.0
+version: 8.10.0
 appVersion: 2.2.1
```

Comment:
will need a rebase soon, once the PR #221 is merged to match the appVersion. You can set it now already

Reply:
Hi @SantoDE, this PR is older and is up to date after that #221. I think that the logical is to integrate this PR first and rebase in the other.

Reply:
#221 is now merged, so please one last rebase :-)